src/components/TrapFocus/tests/TrapFocus.test.tsx

```diff
@@ -257,15 +245,15 @@ describe('<TrapFocus />', () => {
 
       listenerMap.keydown({
         keyCode: Key.Tab,
-        target: lastNode(trapFocus),
+        target: trapFocus.find('input')!.domNode,
```

Comment:
This (and a few other instances) slightly diverges from the intent of the test but I feel good about this change.